src/System.Data.SqlClient/src/Interop/SNINativeMethodWrapper.Unix.cs

```diff
@@ -20,6 +20,14 @@ internal enum SniSpecialErrors : uint
             // max error code value
             MaxErrorValue = SNICommon.MaxErrorValue
         }
+
+        internal static int SniMaxComposedSpnLength
```

Comment:
This is not needed for Managed SNI. SniMaxComposedSpnLength is being referenced while creating the spn buffer. In Managed Code we don't have to initialize the buffer. It's initialized when the SPN is computed and assigned in the SNITcpHandle.